css/rules.css

```diff
@@ -0,0 +1,27 @@
+.main__container {
+  width: 100%;
+  height: 100vh;
+}
+
+.container__rules {
+  width: 80rem;
+  height: 60rem;
+  /* background-color: var(--primary-shade); */
```

Comment:
I guess you do not need this bg color, so why not remove it.

Reply:
okay i'll do that.